packages/ios_platform_images/example/ios/RunnerTests/IosPlatformImagesTests.swift

```diff
@@ -0,0 +1,107 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import Flutter
+import XCTest
+
+@testable import ios_platform_images
+
+class IosPlatformImagesTests: XCTestCase {
+
+  var mockChannel: MockMethodChannel!
+
+  override func setUp() {
+    super.setUp()
+    mockChannel = MockMethodChannel()
+  }
+
+  func testHandleMethodCall_loadImage() {
+    let name = "flutter"
+
```

Comment:
Why is there so much vertical whitespace in these files? Setup can be a single logical block.